_data/steps.yml

```diff
@@ -1,20 +1,48 @@
-
-- 5:
-  title: 'Improve Repository Directory Caching'
-  description: 'When running builds, directory caching should be fast, reliable and smart. We have deployed changes to use gzip vs bzip, and will be changing from mtime detection to md5 checksums.'
+- 17:
+  title: 'Enterprise: Trusty build containers'
+  description: 'Provide Ubuntu 14.04 (Trusty Tar) build images for Enterprise users.'
```

Comment:
Ahh here it's `Trusty Tahr`. I believe I also mispelled it in my comment in that other ticket earlier. Sorry! 😞